### PR TITLE
Add theme docs to style component section

### PR DESCRIPTION
**Pull request by beth-panx**

### CODE REVIEW

File: concepts/toolkit/customize-components/style.md [hunk 1]
```diff
@@ -2,14 +2,14 @@
 title: "Styling the Microsoft Graph Toolkit"
 description: "Use CSS custom properties to modify the component styles"
 localization_priority: Normal
-author: nmetulev
+author: nmetulev, beth-panx
```
Comment (beth-panx) [inline concepts/toolkit/customize-components/style.md:5]: I am actually not sure if this is the correct way of doing multiple authors :D

Comment (nmetulev) [reply concepts/toolkit/customize-components/style.md:5]: I guess we'll find out when we do the PR to master :)

File: concepts/toolkit/customize-components/style.md [hunk 2]
```diff
@@ -2,14 +2,14 @@
...
 ---
 
 # Styling components in the Microsoft Graph Toolkit
 
-Use CSS custom properties to modify the component styles.
+## Use CSS custom properties
 
-Each component documents a set of [CSS custom properties](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties) that you can use to change the look and feel of certain elements. For example:
+Each component documents a set of [CSS custom properties](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties) that you can use to change the look and feel of certain elements. You can find the available custom CSS properties in each component docs. For example:
```
Comment (nmetulev) [inline concepts/toolkit/customize-components/style.md:12]: Are there any updates to the css properties for each component that should also be added as part of this PR? Or are all the properties the same?

File: concepts/toolkit/customize-components/style.md [hunk 3]
```diff
@@ -21,6 +21,50 @@ You can't style internal elements of a component unless you provide a CSS custom
...
 For more flexibility, consider using [custom templates](./templates.md).
 
+## Apply themes
+
+There are two themes available - light and dark. By default, all components are styled with light theme. To switch to dark theme, you can simply apply `class="mgt-dark"` to the desired section of your HTML page. The components inside of that sections will have dark theme apply to them. Here are some examples of how themes will apply based on how you structure your HTML:
+
+Example 1: Global theme
+```html
+<body class="mgt-light">
+    // light theme will apply to all components in this section
```
Comment (nmetulev) [inline concepts/toolkit/customize-components/style.md:31]:
```suggestion
    <!-- light theme will apply to all components in this section -->
```

File: concepts/toolkit/customize-components/style.md (hunk 2 above)
Comment (beth-panx) [reply concepts/toolkit/customize-components/style.md:12]: That's a great question! I have to a comparison. I think I may have changed some names...

Comment (beth-panx) [reply concepts/toolkit/customize-components/style.md:12]: Added @nmetulev

File: concepts/toolkit/customize-components/style.md (hunk 1 above)
Comment (nmetulev) [inline concepts/toolkit/customize-components/style.md:5]: Apparently the build bot doesn't like this :(

```suggestion
author: beth-panx
```

**Review by nmetulev — APPROVED** [review]

**Pull request merged**